### ISSUE TITLE
Remove Null Checks on PermissonHolder in Services

### ISSUE THREAD

**Issue by terrypacker** [infiniteautomation/ma-core-public#1572]
This code can be removed as the PermissionService will pick that up now.

```
PermissionHolder user = Common.getUser();
Objects.requireNonNull(user, "Permission holder must be set in security context");
```

**Comment by jazdw** [comment]
- User should no longer be null, but an instance of Anonymous
- Don't throw NPE but PermissionException

**Comment by jazdw** [comment]
Also ensure we do not check if a user is null anywhere as a part of access control.

**Comment by terrypacker** [comment]
Remove the `runAs...` methods from the service layer and assume anytime the SecurityContext permission holder is null that we run with superadmin permission.  See #1624

**Comment by jazdw** [comment]
Fixed via e27a3389ede6aaddc17f52956378fc9f116792c2
